cms/app/assets/tailwind/application.css

```
@@ -0,0 +1,278 @@
+@import "tailwindcss";
+@source "../../views";
```

Stylelint (17.2.0)

[error] Unexpected unknown at-rule "@source" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)